src/js/components/Map.js

```diff
@@ -153,21 +154,27 @@ export default class Map extends Component {
       if (!editingEnabled) {
         editToolbar.deactivate();
       } else {
-        if (map.infoWindow && map.infoWindow.getSelectedFeature) {
+        if (map.infoWindow && map.infoWindow.getSelectedFeature()) {
           const selectedFeature = map.infoWindow.getSelectedFeature();
+          console.log('selectedFeature', selectedFeature);
           if (selectedFeature && selectedFeature.geometry) {
             editToolbar.activate(Edit.EDIT_VERTICES, selectedFeature);
           }
         }
+        return;
```

Comment:
@KaylaKremer Why the `return` statement?

Reply:
Accidentally left in. Will remove.

Reply:
Accidentally left in. Will remove.

Reply:
@KaylaKremer Coolio - after that, let's resolve the **conflicts** and then merge this in